modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-configuring-live-migration-policies.adoc
+
+
+:_content-type: PROCEDURE
+[id="virt-configuring-a-live-migration-policy_{context}"]
+= Configuring a live migration policy
+
+Use the `MigrationPolicy` custom resource definition (CRD) to define migration policies for one or more groups of selected virtual machine instances (VMIs).
+
+You can specify groups of VMIs by using any combination of the following:
+
+* Virtual machine (VM) labels such as `size`, `os`, `gpu`, and other VM labels.
```

Comment:
There are a few more instances of `VM` instead of `VMI` in this file. I would suggest to replace it to be 100% clear that we're operating on VMIs here

Reply:
Ah I assumed VM label was different context - will update!